python/triton/experimental/gluon/_runtime.py

```diff
@@ -17,16 +16,13 @@ def __init__(self, fn, signature, constexprs=None, attrs=None) -> None:
         self.ext = "ttgir"
 
     def make_ir(self, options, codegen_fns, module_map, context):
-        from triton.compiler.compiler import make_backend
         from triton.compiler.code_generator import ast_to_ttir
 
         builder = ir.builder(context)
         module = builder.create_module()
 
         # Assign module attributes eagerly, as they are needed to verify layouts
-        target = triton.runtime.driver.active.get_current_target()
-        backend = make_backend(target)
-        target = backend.get_target_name(options)
+        target = f"{options.backend_name}:{options.arch}"
```

Comment:
I don't think this is right, the cuda implementation of `get_target_name` is expecting to go from e.g. `sm80` -> `cuda:80`.

Reply:
Okay, I peeled this out into https://github.com/triton-lang/triton/pull/7689 given it's effectively separate.